library/lua/dfhack.lua

```diff
@@ -695,7 +695,14 @@ local valid_script_flags = {
     scripts = {required = false},
 }
 
+local warned_scripts = {}
 function dfhack.run_script(name,...)
+    if not warned_scripts[name] and require('helpdb').get_entry_tags(name).untested then
```

Comment:
I would put this in `run_script_with_env` so it fires if a script is `enable`d as well.

Reply:
done